src/rustllvm/RustWrapper.cpp

```diff
@@ -1219,11 +1219,17 @@ extern "C" void LLVMRustFreeOperandBundleDef(OperandBundleDef *Bundle) {
 extern "C" LLVMValueRef LLVMRustBuildCall(LLVMBuilderRef B, LLVMValueRef Fn,
                                           LLVMValueRef *Args, unsigned NumArgs,
                                           OperandBundleDef *Bundle,
+                                          unsigned NoTail,
                                           const char *Name) {
   unsigned Len = Bundle ? 1 : 0;
   ArrayRef<OperandBundleDef> Bundles = makeArrayRef(Bundle, Len);
-  return wrap(unwrap(B)->CreateCall(
-      unwrap(Fn), makeArrayRef(unwrap(Args), NumArgs), Bundles, Name));
+  CallInst *call = unwrap(B)->CreateCall(unwrap(Fn),
+                                         makeArrayRef(unwrap(Args), NumArgs),
+                                         Bundles,
+                                         Name);
+  if (NoTail)
+    call->setTailCallKind(CallInst::TCK_NoTail);
```

Comment:
Can you make a separate function that calls `setTailCallKind`?

Reply:
What is the benefit of that?

Reply:
Not having random boolean arguments in the API, both in C++ and Rust.

Reply:
I added a separate function to mark CallInsts as notail.